modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc

```diff
@@ -39,11 +39,11 @@ For more information about features, see xref:identity-management.adoc[{keycloak
 
 . To demonstrate {unifiedpush-service}, xref:push-notifications.adoc#provisioning-push-notifications-service[Provision the {unifiedpush-service} service]. For more information about features, see xref:push-notifications.adoc[{unifiedpush-service} Service]
 
-// tag::excludeDownstream[]
+// tag::excludeUpstream[]
 
-NOTE: You can only demonstrate {sync-service} on Cordova.
+NOTE: You can only demonstrate {sync-service} on Cordova or Ionic.
```

Comment:
Hi @finp, my understanding is that the data sync can be built upon a Cordova or Ionic platform, and as such, I've included *Ionic* here - make sense to you?

Reply:
There are a few things here:
- a terminology question: Ionic is cordova, or at least our support for Ionic requires cordova, which is why we need to say 'Plain Cordova'
- I don't think this sentence and tagging make sense anymore... might be best to remove, create a jira to follow up?

Reply:
That makes sense :+1:

Reply:
https://issues.jboss.org/browse/AEROGEAR-9371